runtime/frequency/src/lib.rs

```diff
@@ -1058,6 +1080,8 @@ construct_runtime!(
 		Msa: pallet_msa::{Pallet, Call, Storage, Event<T>} = 60,
 		Messages: pallet_messages::{Pallet, Call, Storage, Event<T>} = 61,
 		Schemas: pallet_schemas::{Pallet, Call, Storage, Event<T>, Config} = 62,
+		Capacity: pallet_capacity::{Pallet, Call, Storage, Event<T>} = 63,
+		FrequencyTxPayment: pallet_frequency_tx_payment::{Pallet, Call, Event<T>} = 64,
```

Comment:
Should this reuse the index of the old Tx pallet?

Reply:
looking into this.

Reply:
The TransactionPayment pallet is used for its [runtime RPC APIs](https://github.com/LibertyDSNP/frequency/blob/2c2d3558391e353bad174833a000c32660fa9fb6/runtime/frequency/src/lib.rs#L1186), for query_fees and query_info functions. 

In theory you could replace it by implementing equivalent RPCs in FrequencyTxPayment and have it call into TransactionPayment::query_info (etc). However to get that to work, you'd need a FrequencyTxPayment Config that is a TransactionPaymentProvider so you could call those functions. But then you’d have to have a runtime Config for TransactionPayment anyway.  You could also completely duplicate all that functionality in FrequencyTxPayment, but that would create a lot of risk and rattle. Unless we want to eliminate those RPC endpoints entirely (I'm assuming we don't),  I think it's best to leave it in there.